tests/integration/conftest.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Pytest configuration for integration tests."""
+
+from time import sleep
+
+import pytest
+
+from invenio_search import current_search_client as es
+
+from inspirehep.factory import create_app
+
+
+@pytest.yield_fixture(scope='session', autouse=True)
+def app(request):
+    """Flask application fixture."""
+    app = create_app()
+
+    with app.app_context():
+        # XXX(jacquerie): import must be local, otherwise task defaults to pickle serializer.
+        from inspirehep.modules.migrator.tasks import add_citation_counts, migrate
+
+        _db = app.extensions['sqlalchemy'].db
+        for table in _db.metadata.sorted_tables:
+            table.create(bind=_db.engine, checkfirst=True)
+
+        # XXX(jacquerie): makes sure that ES is up.
+        sleep(10)
+
+        _es = app.extensions['invenio-search']
+        list(_es.create(ignore=[400]))
+
+        migrate('./inspirehep/demosite/data/demo-records.xml.gz', wait_for_results=True)
+
+        # XXX(jacquerie): makes sure that all HEP records were migrated.
```

Comment:
Ah, yeah, that is a notation that I use when developing, no need to commit that. The comment is enough.